### DISCUSSION TITLE
Import & Export functions don't work

### DISCUSSION THREAD

**Discussion by pnthach95** [valinet/ExplorerPatcher#1120] [General]
When press button, Properties window crashes(?)

- Windows 11 22H2 build 22621.1413
- ExplorerPatcher version 22621.1413.54.3

https://user-images.githubusercontent.com/31266357/226153102-7c775db8-7826-4ba4-b7eb-efa708d0dc95.mp4

And you should have a button to copy version number, it's long

**Comment by valinet** [comment]
I cannot reproduce. Collect a crash dump so we can see exactly where it crashes, then we can determine what setting triggers it. Thanks.

**Reply by pnthach95** [reply]
Hi, I got you dump files, include import and export
[explorerpatcher_dumps.zip](https://github.com/valinet/ExplorerPatcher/files/11010784/explorerpatcher_dumps.zip)

**Reply by valinet** [reply]
It shows exception on `d3d11!CModule::GetCompatValue+61` which I don't even remotely touch... Yeah, idk, the crash dumps seem not to be full crash dumps, so there's limited info in there.

**Reply by pnthach95** [reply]
I pressed `Import settings` and it generated dumps with different size. Is it normal?

![image](https://user-images.githubusercontent.com/31266357/226183755-83e246e0-a767-4499-8a2f-e79d261a96f2.png)
